public/about.html

```diff
@@ -33,4 +33,11 @@
       </div>
     </div>
   </body>
+  <footer>
+    <div class="container giphy">
```

Comment:
I left the div with class container because it appears to be inheriting font styling from somewhere.

with container:
<img width="721" alt="about__and_about_html_ ___downloads" src="https://cloud.githubusercontent.com/assets/9340454/20820747/6406d582-b7f2-11e6-8af5-d977ce948f3c.png">

without container:
<img width="640" alt="about__and_about_html_ ___downloads 2" src="https://cloud.githubusercontent.com/assets/9340454/20820748/68f9cf9a-b7f2-11e6-8550-626617fbbc94.png">